### DISCUSSION TITLE
Hitbox position does not follow sprite position

### DISCUSSION THREAD

**Discussion by EzzypooOfNazareth** [HaxeFlixel/flixel#2338] [Q&A] [answered]
I have created a sprite that is supposed to act as the floor of an infinite scrolling platformer. The camera is constantly scrolling to the right and I update the position of the floor sprite based of the FlxG.camera.scroll.x. However, this does not change the hitbox's position even when calling the updateHitbox() function within the update loop. What am I doing wrong?

```
groundCollisionGroup.forEach(function(gc)
{
	gc.x = FlxG.camera.scroll.x; // updates sprite position

	gc.updateHitbox(); // update hitbox, which does not change account for sprite position
});
```

**Comment by EzzypooOfNazareth** [comment, marked as answer]
I've since found the answer at https://stackoverflow.com/questions/43792751/flxg-collide-not-working-when-x648

**Comment by NulllStack** [comment]
There's another way to reset the worldBounds if you're using the `FlxTilemap`, you could just use `tilemap.follow()` and the collision quad tree will fit the tile map's size.

**Reply by NulllStack** [reply]
BTW, it'll also change the cameras behavior

**Reply by NulllStack** [reply]
https://api.haxeflixel.com/flixel/tile/FlxTilemap.html
The function call reset the collision quad tree by default, you could check the source code to find out how it work
